Firestore/Source/Public/FirebaseFirestore/FIRTimestamp.h

```diff
@@ -59,15 +59,15 @@ NS_SWIFT_NAME(Timestamp)
 /** Creates a new timestamp with the current date / time. */
 + (instancetype)timestamp;
 
-/** Returns a new NSDate corresponding to this timestamp. This may lose precision. */
+/** Returns a new `Date` corresponding to this timestamp. This may lose precision. */
 - (NSDate *)dateValue;
 
 /**
  * Returns the result of comparing the receiver with another timestamp.
  * @param other the other timestamp to compare.
- * @return NSOrderedAscending if `other` is chronologically following self,
- *     NSOrderedDescending if `other` is chronologically preceding self,
- *     NSOrderedSame otherwise.
+ * @return `NSOrderedAscending` if `other` is chronologically following self,
+ *     `NSOrderedDescending` if `other` is chronologically preceding self,
+ *     `NSOrderedSame` otherwise.
```

Comment:
Looks like the Swift names are different: https://developer.apple.com/documentation/foundation/comparisonresult

Reply:
oops. Done.